sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -92,6 +92,72 @@ protected CommunicationIdentityClient()
             RestClient = null;
         }
 
+        /// <summary>Gets user details. <see cref="CommunicationUserIdentifier"/>.</summary>
+        /// <param name="communicationUser">Identifier of the identity. </param>"
+        /// <param name="cancellationToken">The cancellation token to use.</param>
+        public virtual Response<CommunicationIdentityResult> GetUser(CommunicationUserIdentifier communicationUser, CancellationToken cancellationToken = default)
```

Comment:
Maybe we can rename the method naming to GetUserInfo or GetUserDetails, or other naming which describes better?

Reply:
I think the GetUser is a clear and concise name, its also contextually appropriate (part of CommunicationIdentityClient) and its matching the other namings: CreateUser/DeleteUser/etc.
Wdyt?